### PR TITLE
fix(frontend): some translation and wording issues

### PR DESCRIPTION
**Pull request by grignards**
See #4114.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link): translations are never tested automatically
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by raphj — APPROVED** [review]

File: frontend/i18next.scanner/pt/translation.json [hunk 1]
```diff
@@ -334,7 +334,10 @@
   "My profile": "Meu perfil",
   "Error while saving public profile": "Erro ao salvar o perfil público",
   "Personal Page": "Página pessoal",
-  "{{ authoredContentRevisionsCount }} interventions in {{ authoredContentRevisionsSpaceCount }} spaces": "{{ authoredContentRevisionsCount }} intervenções em {{ authoredContentRevisionsSpaceCount }} espaços",
+  "{{ count }} intervention ": "{{ count }} intervenção ",
+  "{{ count }} contribution _plural": "{{ count }} intervençãos ",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:338]:
```suggestion
  "{{ count }} contribution _plural": "{{ count }} intervenções ",
```

File: frontend/src/component/PublicProfile/Information.jsx [hunk 2]
```diff
@@ -28,13 +28,8 @@ export const Information = props => {
...
         </div>
         <div>
-          {props.t(
-            '{{ authoredContentRevisionsCount }} interventions in {{ authoredContentRevisionsSpaceCount }} spaces',
-            {
-              authoredContentRevisionsCount: props.authoredContentRevisionsCount,
-              authoredContentRevisionsSpaceCount: props.authoredContentRevisionsSpaceCount
-            }
-          )}
+          {props.t('{{ count }} contribution ', { count: props.authoredContentRevisionsCount })}
```
Comment (gbonaspetti) [inline frontend/src/component/PublicProfile/Information.jsx:31]:
```suggestion
          {props.t('{{ count }} intervention ', { count: props.authoredContentRevisionsCount })}
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

**Comment by gbonaspetti** [comment]
During tests:
- The space activity feed breadcrumbs are not translated
breadFeed

**Review by raphj — APPROVED** [review]

**Comment by raphj** [comment]
I liked "contribution", though

**Pull request merged**